### PR TITLE
reflect internal regulation changes

### PR DESCRIPTION
**Pull request by francisco-core**
The internal regulation changed and these are the related updates

### CODE REVIEW

**Comment by anadahz** [comment]
@francisco-core thanks for working on it.
The file `static/documents/regulamento-interno_v1.pdf` is not being linked anywhere, is this in purpose?

I have converted the PDF to a Hugo page. Now all we need is an appropriate section for the association's documents.

**Comment by francisco-core** [comment]
Done @anadahz. Feel free to merge it.

File: layouts/partials/footer.html [hunk 1]
```diff
@@ -34,33 +32,32 @@ <h5 class="text-left add-bottom footer-link">
...
     </h5>
     <ul class="list-unstyled">
       <li><a class="footer-link" href="https://bbb.privacylx.org" target="_blank" >BigBlueButton</a></li>
-      <li><a class="footer-link" href="https://github.com/privacylx/devops" target="_blank" >Ansible Infrastructure</a></li>
-      <li><a class="footer-link" href="{{ .Site.Params.ghrepo }}" target="_blank" >Website Source Code</a></li>
-      <li><a class="footer-link" href="{{ .Site.Params.bugtracker }}" target="_blank">Bugs & Suggestion</a></li>
-
+      <li><a class="footer-link" href="https://github.com/privacylx/devops" target="_blank" >{{ i18n "devops" }}</a></li>
+      <li><a class="footer-link" href="{{ .Site.Params.ghrepo }}" target="_blank" >{{ i18n "website-source" }}</a></li>
+      <li><a class="footer-link" href="{{ .Site.Params.bugtracker }}" target="_blank">{{ i18n "issue-tracker" }}</a></li>
```
Comment (francisco-core) [inline layouts/partials/footer.html:42]: great! You fixed my laziness! Thx

File: themes/hugo-bootstrap/i18n/en.yaml [hunk 2]
```diff
@@ -190,3 +190,12 @@
 
 - id: resources
   translation: "Resources"
+
+- id: devops
+  translation: "Our Infrastructure As Code"
```
Comment (francisco-core) [inline themes/hugo-bootstrap/i18n/en.yaml:195]: This is too long. It occupies 2 lines. We should make it "Infrastructure code"

File: themes/hugo-bootstrap/i18n/pt.yaml [hunk 3]
```diff
@@ -189,3 +189,12 @@
 
 - id: resources
   translation: "Recursos Educativos"
+
+- id: devops
+  translation: "A Nossa Infraestrutura Como Código"
```
Comment (francisco-core) [inline themes/hugo-bootstrap/i18n/pt.yaml:194]: "Infraestrutura" to make it shorter.

File: themes/hugo-bootstrap/i18n/en.yaml [hunk 4]
```diff
@@ -190,3 +190,12 @@
...
+
+- id: website-source
+  translation: "Website Código Fonte"
```
Comment (francisco-core) [inline themes/hugo-bootstrap/i18n/en.yaml:198]: "Website Source" to make it shorter <-- remember this is the "en.yaml" not "pt.yaml"

File: themes/hugo-bootstrap/i18n/en.yaml [hunk 5]
```diff
@@ -190,3 +190,12 @@
...
+
+- id: issue-tracker
+  translation: "Sugestões & Bugs"
```
Comment (francisco-core) [inline themes/hugo-bootstrap/i18n/en.yaml:201]: "Suggestions & Bugs" <--- this is the EN file

File: themes/hugo-bootstrap/i18n/pt.yaml [hunk 6]
```diff
@@ -189,3 +189,12 @@
...
+
+- id: website-source
+  translation: "Website Source Code"
```
Comment (francisco-core) [inline themes/hugo-bootstrap/i18n/pt.yaml:197]: "Website Source" <-- shorter.

Hard to translate in short words

File: themes/hugo-bootstrap/i18n/pt.yaml [hunk 7]
```diff
@@ -189,3 +189,12 @@
...
+
+- id: issue-tracker
+  translation: "Suggestions & Bugs"
```
Comment (francisco-core) [inline themes/hugo-bootstrap/i18n/pt.yaml:200]: "Sugestões e Bugs" <-- PT not EN

**Review by francisco-core — COMMENTED** [review]
Thanks for taking a look at this and fixing my laziness. I left a few comments

**Comment by anadahz** [comment]
Thanks @francisco-core for the review, please see my fix in 7fa7b3a .

**Comment by francisco-core** [comment]
> Your [Render](https://render.com) PR Server URL is https://privacylx-org-pr-53.onrender.com.
> 
> Follow its progress at https://dashboard.render.com/static/srv-bukoil2pp1juo912rrng.

@anadahz were you the one enabling this bot, or is this spam?

**Comment by francisco-core** [comment]
One of my commits improves the small screen rendering:

## Before
without

## After
with

**Comment by francisco-core** [comment]
@anadahz it's now ready to be merged. Thanks for the review.

**Comment by anadahz** [comment]
> > Your [Render](https://render.com) PR Server URL is https://privacylx-org-pr-53.onrender.com.
> > Follow its progress at https://dashboard.render.com/static/srv-bukoil2pp1juo912rrng.
> 
> @anadahz were you the one enabling this bot, or is this spam?

This is a feature I enabled. It builds automatically the website on every pull request.
This way we can visually inspect potential issues of the changes.

Thanks for the improvements and review, I'm going to merge this PR now.

**Pull request merged**